app/containers/Products/ProductRowItemContainer.js

```diff
@@ -12,55 +15,8 @@ import {Form, Button, ButtonGroup, Col, Row, Modal} from 'react-bootstrap';
 
 // TODO: The UI is a little borked, the edit buttons apply to all items on the page because of where state lives currently
 //       I have purposely left this not fixed as I believe this should be fixed in a separate refactor of this page
-export const ProductRowItemContainer = props => {
-  const createBenchmark = graphql`
-    mutation ProductRowItemContainerBenchmarkMutation(
-      $input: CreateBenchmarkInput!
-    ) {
-      createBenchmark(input: $input) {
-        benchmark {
-          rowId
-        }
-      }
-    }
-  `;
-  const createProduct = graphql`
-    mutation ProductRowItemContainerProductMutation(
-      $input: CreateProductInput!
-    ) {
-      createProduct(input: $input) {
-        product {
-          rowId
-        }
-        query {
-          ...ProductListContainer_query
-        }
-      }
-    }
-  `;
-  const updateBenchmark = graphql`
-    mutation ProductRowItemContainerUpdateBenchmarkMutation(
-      $input: UpdateBenchmarkByRowIdInput!
-    ) {
-      updateBenchmarkByRowId(input: $input) {
-        benchmark {
-          rowId
-        }
-      }
-    }
-  `;
-  const updateProduct = graphql`
-    mutation ProductRowItemContainerUpdateProductMutation(
-      $input: UpdateProductByRowIdInput!
-    ) {
-      updateProductByRowId(input: $input) {
-        product {
-          rowId
-        }
-      }
-    }
-  `;
 
+export const ProductRowItemContainer = props => {
```

Comment:
Having a named export that is the same name as the file will trigger a linter error when you write 
```js
import ProductRowItemContainer from 'containers/Products/ProductRowItemContainer';
```
Rename to `ProductRowItemComponent` and/or rename the file to `ProductRowItem` ?  It's in the `containers` folder already, no need to append `Container` to the name

Reply:
A review & shared understanding on naming conventions would be helpful for everyone on the team. I think we are all doing different things.

Reply:
Changing the name of the function broke all sorts of stuff, possibly to do with the export / default export we are doing in this file. I've changed it back for now & maybe we can revisit how to change these after we have defined a style guide.